src/zope/securitypolicy/metaconfigure.py

```diff
@@ -27,9 +27,9 @@
 
 
 def grant(_context, principal=None, role=None, permission=None):
-    nspecified = ((principal is not None)
-                  + (role is not None)
-                  + (permission is not None))
+    nspecified = ((principal is not None) +
+                  (role is not None) +
+                  (permission is not None))
```

Comment:
PEP-8 recommends operators on the new line: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

This was changed pretty recently (PEP-8 is a living document), and the `pep8` tool on PyPI (or at least the older version pulled in by `flake8`) doesn't understand that.